wasatch/FeatureIdentificationDevice.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, device_id: str, message_queue: list = None) -> None:
         self.message_queue = message_queue
 
         self.device = None
-        if "MOCK" in str(device_id):
+        if "MOCK" in str(device_id).upper():
```

Comment:
Does this need to use .upper? Looking at the other DeviceIDs they're already upper case (FILE, USB), so I'm thinking MOCK should be upper case already and Mock would be an invalid DeviceID.

Reply:
Well, without this change, all release tests now fail.  When I put a debug line in the `else` block, I get this:
```
FID.ctor: device_id was <MockUSBDevice 0x731d4:0x4000:111111:111111> so device_type is RealUSBDevice
```